src/repository.ts

```diff
@@ -213,6 +213,19 @@ class RepositorySourceControlManager {
 
     this.parentResourceGroups = updatedGroups;
 
+    const trackedFilesList = await this.repository.fileList();
+    const trackedFiles = new Set<string>();
+
+    for (const t of trackedFilesList) {
+      const pathParts = t.split('/');
```

Comment:
I'm worried using forward slash here and on the line below won't be safe on Windows, @kevin314 would you mind testing this on your Windows machine?

Reply:
You are right. I will try to use https://nodejs.org/api/path.html#pathsep to fix this.

Reply:
I used `path.sep`.